### DISCUSSION TITLE
Reorder image condition and clicking x distance from condition

### DISCUSSION THREAD

**Discussion by tosamyng** [Nain57/Smart-AutoClicker#303] [General Feature Requests]
Hi is it possible to implement a function that allow us to instead of clicking on a condition or on a set coordinate but instead a set distance from the condition? Im thinking it would have 2 variables, an angle and a distance. Make the angles like the unit circle.

The 2nd thing thing I did like is to be able to reorder my image conditions so I dont have to copy and paste and delete the old one.

**Comment by Nain57** [comment]
Closed as implemented (a long ago !)
You can check the wiki: https://github.com/Nain57/Smart-AutoClicker/wiki
